actionpack/lib/action_controller/test_case.rb

```diff
@@ -491,58 +491,64 @@ def determine_default_controller_class(name)
         end
       end
 
-      # Simulate a GET request with the given parameters.
+      # Simulate a GET request with the given keyword arguments.
       #
-      # - +action+: The controller action to call.
-      # - +parameters+: The HTTP parameters that you want to pass. This may
-      #   be +nil+, a hash, or a string that is appropriately encoded
+      # - action: The controller action to call.
+      # - params: The hash with HTTP parameters that you want to pass. This may be +nil+.
+      # - body: The request body with a string that is appropriately encoded
       #   (<tt>application/x-www-form-urlencoded</tt> or <tt>multipart/form-data</tt>).
       # - +session+: A hash of parameters to store in the session. This may be +nil+.
```

Comment:
@kirs when you refer to variables, keyword arguments, etc. you use fixed-width font for them. That is `assigns`, rather than just assigns. Have a look at the [API guidelines](http://edgeguides.rubyonrails.org/api_documentation_guidelines.html#fonts).